compute/main.py

```diff
@@ -0,0 +1,46 @@
+import os
+import yaml
+from compute.engine import ComputeEngine
+from compute.rabbitmq_broker import RabbitMQBroker
+from detectors.mocks import MockDetector
+
+
+def load_config(file_path: str) -> dict:
+    with open(file_path, "r") as file:
+        return yaml.safe_load(file)
```

Comment:
Also need mechanism to load from environment variables - vars like `RABBITMQ_HOST` should be resolved accordingly